distribution/ddf-common/src/main/resources/etc/startup.properties

```diff
@@ -0,0 +1,10 @@
+# Bundles to be started on startup, with startlevel
+mvn\:org.apache.sshd/sshd-core/0.14.0 = 20
+mvn\:org.apache.felix/org.apache.felix.metatype/1.1.2 = 5
+mvn\:org.apache.karaf.services/org.apache.karaf.services.eventadmin/4.0.4 = 5
```

Comment:
Should use ${karaf.version}, ${pax.logging.version} and whatever other variables we already have declared for these version